src/main/java/core/mate/academy/impl/ExcavatorProducer.java

```diff
@@ -0,0 +1,19 @@
+package core.mate.academy.impl;
+
+import core.mate.academy.model.Excavator;
+import core.mate.academy.service.MachineProducer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer<Excavator> {
+    private List<Excavator> list = new ArrayList<>();
```

Comment:
The field 'list' should have an access modifier. Consider using 'private' to encapsulate the data if it is not intended to be accessed from outside the class.